Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesInterface.cpp

```diff
@@ -356,6 +356,16 @@ namespace AzFramework
         return *this;
     }
 
+    bool EntitySpawnTicket::operator==(const EntitySpawnTicket& rhs) const
+    {
+        return GetId() == rhs.GetId();
+    }
+
+    bool EntitySpawnTicket::operator!=(const EntitySpawnTicket& rhs) const
+    {
+        return GetId() != rhs.GetId();
```

Comment:
I think you can do this in the situation where the equality becomes more complex. on line 359.
```suggestion
        return !(*this == rhs);
```

Reply:
good point, thanks